### PR TITLE
feat(web) :sparkles: new config to hide login button on WaitForOwnerDialog

### PR DESCRIPTION
**Pull request by HannesOberreiter**
If the guest joins before the host and jitsi is setup with JWT only authentication the WaitForOwnerDialog shows a "Login" button, which is without use in this case.

- Added a config option to hide the login button showing up.
- Added new text if login button is hidden

Translations are added for EN, DE, HU. Left empty string for the other translations.

`config.hideLoginButton=false`
config.hideLoginButton=false

`config.hideLoginButton=true`
config.hideLoginButton=true

A relevant discussion/question I found in the community forum: https://community.jitsi.org/t/how-to-disable-login-pop-up-along-with-supporting-jwt-authentication/110517

Contribution Agreement was signed as company, Certible GmbH.

Cheers
Hannes

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1776,6 +1776,9 @@ var config = {
     //         collectionInterval?: number;
     //         logGetStats?: boolean;
     // },
+
+    // Hide login button on auth dialog, you may want to enable this if you are using JWT tokens to authenticate users
+    hideLoginButton: true,
```
Comment (saghul) [inline config.js:1781]: Please comment this out, it's not a good default.

**Review by saghul — CHANGES_REQUESTED** [review]

File: config.js (hunk 1 above)
Comment (HannesOberreiter) [reply config.js]: Of course sorry about that.

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by HannesOberreiter** [comment]
Updated conflicts with translation files.

**Review by saghul — APPROVED** [review]
LGTM! @damencho note for ourselves: perhaps we should conisder adding an `auth` section to config.js...

**Review by damencho — APPROVED** [review]

**Pull request merged**